app/lib/consented_criteria.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ConsentedCriteria
```

Comment:
I think the use of this class for the emails means we need to make some further changes. For example, should the consent status of a patient also use this method to ensure that when the year rolls over the patient is no longer appearing as having received consent for Flu?

I would suggest we keep the scope of this PR specific to vaccinations, and we worry about the consent expiration issue when we deal with rollover (#1943). Fixing the issue for previous vaccinations solves the immediate problem as we won't have any patients with previous consents for Flu for the programme this coming year.